src/util/FactorUtil.js

```diff
@@ -285,6 +285,21 @@ function (Okta) {
     return result.join(' ');
   };
 
+  fn.getCardinalityText = function (enrolled, required, cardinalityInfo) {
+    if (enrolled) {
+      return (cardinalityInfo.enrolled === 1) ? false :
+        Okta.loc('enroll.choices.cardinality.setup', 'login', [cardinalityInfo.enrolled]);
+    }
+    else if (required) {
```

Comment:
I'm not sure.. The mocks have 2 required, i guess so that it's obvious how many are required before you start setup.. @vincentvoong-okta input please?

Reply:
@gowthamidommety-okta @mauriciocastillosilva-okta 

Is "Required" already shown in the title view? If so, I'm okay with changing it to "0 of 2 set up". The mock was designed with the notion that users didn't know they were required.

Reply:
@vincentvoong-okta The heading says `Setup required`

<img width="415" alt="screen shot 2018-10-25 at 3 48 04 pm" src="https://user-images.githubusercontent.com/26014318/47534752-69a11780-d86d-11e8-9208-7f21ed0cdd64.png">

Reply:
Okay, in that case I think its fine if we change it to "0 of 2 set up". We should also change the title to "Set up required" (two words).